node/pkg/fetcher/fetcher.go

```diff
@@ -158,7 +159,7 @@ func (f *Fetcher) startAdapterById(ctx context.Context, adapterId int64) error {
 	return errors.New("adapter not found")
 }
 
-func (f *Fetcher) stopAdapter(ctx context.Context, adapter *AdapterDetail) error {
+func (f *Fetcher) stopAdapter(adapter *AdapterDetail) error {
```

Comment:
@nick-bisonai This is just minor semantical comment, but `adapter` is not meant to be something that it is stopped or started. It does not have state, it only [describes data](https://config.orakl.network/adapter/ada-usdt.adapter.json) in a form that fetcher can consume and understand how to fetch them using its definition. 

For this reason, I think it would be more appropriate to call it `fetcher.stop` and `fetcher.start` than `fetcher.stopAdapter` and `fetcher.startAdapter`. The same applies to function with the `ById` suffix.

Reply:
I was taking it a bit differently, I thought adapter as a component of fetcher. so I thought fetcher.stop will stand for stopping whole operation of fetcher rather than certain adapter of fetcher.

in a same way, I'm also having same consideration about aggregator.
for both fetcher and aggregator, it both has higher dimensional object which contains its components. fetcher -> adapter/ aggregator -> ?

it needs both operations for each elements and as a whole. and it's quite difficult for me to find appropriate naming to easily tell difference between them and easily understand the structure

Reply:
Okay! We can discuss it in more detail during sync today 🙌